dspace/bin/dspace.bat

```diff
@@ -38,6 +38,9 @@ REM If JAVA_OPTS specified, use those options
 REM Otherwise, default Java to using 256MB of memory
 if "%JAVA_OPTS%"=="" set "JAVA_OPTS=-Xmx256m -Dfile.encoding=UTF-8"
 
+REM Add log4j2 configuration to JAVA_OPTS
+set "JAVA_OPTS=%JAVA_OPTS% -Dlog4j2.configurationFile=%DSPACE_CLASSPATH%\config\log4j2-cli.xml"
```

Comment:
This line is incorrect.  Instead of using `%DSPACE_CLASSPATH%`, it should use the existing `%cd%` variable.

So, this should become:
```
set "JAVA_OPTS=%JAVA_OPTS% -Dlog4j2.configurationFile=%cd%\config\log4j2-cli.xml"
```

I've tested this change on Windows and it works properly. The current code you have throws an error.

Reply:
Thanks for catching this I didn't have access to a windows machine to verify, I resolved this issue now (squashed with last commit)